tajo-core/src/test/java/org/apache/tajo/engine/query/TestJoinQuery.java

```diff
@@ -1191,4 +1192,16 @@ public final void testJoinWithOrPredicates() throws Exception {
     assertResultSet(res);
     cleanupQuery(res);
   }
+
+  @Test
+  public final void testNaturalJoin() throws Exception {
+    ResultSet res = null;
+    try {
+      res = executeQuery();
+    } catch (Exception e) {
```

Comment:
This catch block is redundant because the testNaturalJoin method throws Exception. I'll remove it before commit.